ui/StatusQ/src/themepalette.cpp

```diff
@@ -0,0 +1,459 @@
+    t->warningColor3 = StatusColors::getColor("warning_orange", 0.1);
+
+    // Success
+    t->successColor1 = StatusColors::getColor("green3");
+    t->successColor2 = StatusColors::getColor("green3", 0.2);
+    t->successColor3 = StatusColors::getColor("green3", 0.3);
+
+    // Mention
+    t->mentionColor1 = StatusColors::getColor("turquoise3");
+    t->mentionColor2 = StatusColors::getColor("turquoise4", 0.3);
+    t->mentionColor3 = StatusColors::getColor("turquoise4", 0.2);
+    t->mentionColor4 = StatusColors::getColor("turquoise4", 0.1);
+
+    // Pin
+    t->pinColor1 = StatusColors::getColor("orange3");
+    t->pinColor2 = StatusColors::getColor("orange4", 0.2);
+    t->pinColor3 = StatusColors::getColor("orange4", 0.1);
+
+    // Direct (white with varying alpha)
+    t->directColor1 = StatusColors::getColor("white");
+    t->directColor2 = StatusColors::getColor("white", 0.9);
+    t->directColor3 = StatusColors::getColor("white", 0.8);
+    t->directColor4 = StatusColors::getColor("white", 0.7);
+    t->directColor5 = StatusColors::getColor("white", 0.4);
+    t->directColor6 = StatusColors::getColor("white", 0.2);
+    t->directColor7 = StatusColors::getColor("white", 0.1);
+    t->directColor8 = StatusColors::getColor("white", 0.05);
+    t->directColor9 = StatusColors::getColor("white", 0.2);
+
+    // Indirect
+    t->indirectColor1 = StatusColors::getColor("black");
+    t->indirectColor2 = StatusColors::getColor("black", 0.7);
+    t->indirectColor3 = StatusColors::getColor("black", 0.4);
+    t->indirectColor4 = StatusColors::getColor("graphite3");
+
+    // Misc
+    t->miscColor1 = StatusColors::getColor("blue5");
+    t->miscColor2 = StatusColors::getColor("purple");
+    t->miscColor3 = StatusColors::getColor("cyan");
+    t->miscColor4 = StatusColors::getColor("violet");
+    t->miscColor5 = StatusColors::getColor("red2");
+    t->miscColor6 = StatusColors::getColor("orange");
+    t->miscColor7 = StatusColors::getColor("yellow");
+    t->miscColor8 = StatusColors::getColor("green4");
+    t->miscColor9 = StatusColors::getColor("moss2");
+    t->miscColor10 = StatusColors::getColor("brown3");
+    t->miscColor11 = StatusColors::getColor("yellow2");
+    t->miscColor12 = StatusColors::getColor("green6");
+
+    // Other
+    t->neutral95 = QColor(0x06, 0x0F, 0x1F);
+    t->dropShadow = StatusColors::getColor("black", 0.08);
+    t->dropShadow2 = StatusColors::getColor("blue8", 0.02);
+    t->dropShadow3 = StatusColors::getColor("blue8", 0.05);
+    t->backdropColor = StatusColors::getColor("black", 0.4);
+    t->statusFloatingButtonHighlight = StatusColors::getColor("blue4", 0.3);
+    t->statusLoadingHighlight = StatusColors::getColor("white", 0.03);
+    t->statusLoadingHighlight2 = StatusColors::getColor("white", 0.07);
+    t->messageHighlightColor = StatusColors::getColor("blue4", 0.2);
+    t->desktopBlue10 = StatusColors::getColor("darkDesktopBlue10");
```

Comment:
All these ^^ `getColor("name")` should use a string literal of some sort; like:


```suggestion
    t->desktopBlue10 = StatusColors::getColor("darkDesktopBlue10"_L1);
```

Reply:
(also below in this file)